server/etcdserver/server_test.go

```diff
@@ -198,13 +205,18 @@ func TestV2SetClusterVersion(t *testing.T) {
 	defer betesting.Close(t, be)
 	cl := newTestClusterWithBackend(t, []*membership.Member{}, be)
 	cl.SetVersion(semver.New("3.4.0"), api.UpdateCapability, membership.ApplyBoth)
+	cfg := config.ServerConfig{
+		ServerFeatureGate: features.NewDefaultServerFeatureGate("TxnModeWriteWithSharedBuffer", nil),
```

Comment:
```suggestion
		ServerFeatureGate: features.NewDefaultServerFeatureGate("test", nil),
```
since the test has nothing to do with TxnModeWriteWithSharedBuffer